sklearn/linear_model/randomized_l1.py

```diff
@@ -121,31 +121,16 @@ def _make_estimator_and_params(self, X, y):
         """Return the parameters passed to the estimator"""
         raise NotImplementedError
 
-    def get_support(self, indices=False):
+    def _get_support_mask(self, indices=False):
         """Return a mask, or list, of the features/indices selected."""
         check_is_fitted(self, 'scores_')
 
         mask = self.scores_ > self.selection_threshold
         return mask if not indices else np.where(mask)[0]
 
-    # XXX: the two function below are copy/pasted from feature_selection,
-    # Should we add an intermediate base class?
-    def transform(self, X):
-        """Transform a new matrix using the selected features"""
-        mask = self.get_support()
-        X = check_array(X)
-        if len(mask) != X.shape[1]:
-            raise ValueError("X has a different shape than during fitting.")
-        return check_array(X)[:, safe_mask(X, mask)]
-
-    def inverse_transform(self, X):
-        """Transform a new matrix using the selected features"""
-        support = self.get_support()
-        if X.ndim == 1:
-            X = X[None, :]
-        Xt = np.zeros((X.shape[0], support.size))
-        Xt[:, support] = X
-        return Xt
+    def get_support(self, indices=False):
```

Comment:
Actually I removed this since it seemed like SelectorMixin inherits TransformerMixin?
```python
class SelectorMixin(six.with_metaclass(ABCMeta, TransformerMixin)):
```